### ISSUE TITLE
Number of runs

### ISSUE THREAD

**Issue by NaroaCS** [CityScope/VehicleClustering#42]
@agrignard - You mentioned a method to determine the number of runs for each parameter setup.
Did you refer to the method described in 2.4. and 2.6 in this article: https://www.jasss.org/18/4/4.html ?

**Comment by agrignard** [comment]
Yes the section you mention are explaning this concept. Which is more or less related to what we call Stochasticity Sensitivity Analysis
It's also explain in the section 3.1 of this article
https://www.researchgate.net/profile/Patrick-Taillandier/publication/357013109_Using_agent-based_modelling_to_understand_advantageous_behaviours_against_COVID-19_transmission_in_the_built_environment/links/61b7fb334b318a6970dd0435/Using-agent-based-modelling-to-understand-advantageous-behaviours-against-COVID-19-transmission-in-the-built-environment.pdf

The basic idea is to understand impact of the randomness of the simulations and to find a threshold number of replicates beyond which the mean values of your indicators are sufficiently accurate (low standard err
